src/gainmap.c

```diff
@@ -660,6 +655,22 @@ avifResult avifRGBImageComputeGainMap(const avifRGBImage * baseRgbImage,
         }
     }
 
+    // Populate the gain map metadata's headrooms.
+    gainMapMetadata.baseHdrHeadroom = log2f(AVIF_MAX(baseMax, kEpsilon));
+    gainMapMetadata.alternateHdrHeadroom = log2f(AVIF_MAX(altMax, kEpsilon));
+
+    // The gainmap is the log-ratio of the higher-headroom (more HDR) image to the lower-headroom
+    // (less HDR) image. Reverse the gainmap if this is the case.
+    if (baseMax > altMax) {
```

Comment:
I am curious why you changed this check to compare `baseMax` and `altMax` instead of `baseHdrHeadroom` and  `alternateHdrHeadroom`?

Reply:
That was an accident (I was playing with using this instead of the `gainMapMetadata` parameters, to see if it was reasonable to populate them all at the end... this ended up staying by accident). Changed it back.